pkg/master/master.go

```diff
@@ -361,7 +361,9 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 		admissionregistrationrest.RESTStorageProvider{},
 		eventsrest.RESTStorageProvider{TTL: c.ExtraConfig.EventTTL},
 	}
```

Comment:
InstallLegacyAPI above should return an error as well and stop fataling internally